### DISCUSSION TITLE
Sub volume, surround volume, night sound, speech enhancement

### DISCUSSION THREAD

**Discussion by bednim** [punxaphil/custom-sonos-card#14] [Ideas]
Hi, sorry, don't want to sound demanding - do you think you could add controls like:

sub volume and on/off switch,
surrounds volume,
night sound,
speech enhancement

?

Thanks!

**Comment by punxaphil** [comment]
I’d love to, but can’t find anything in the apis that support it. Let me know though if you have any suggestions on what api calls to use

**Reply by bednim** [reply]
hi @johanfrick , over a year has passed, wonder if this could be now implemented? 

you have asked for API - home assistant sonos integration got those options - is this something you could use?

thank you

**Reply by punxaphil** [reply]
Hi! Can you provide links or screenshots that shows it's supported?

**Reply by bednim** [reply]
sure - this is screen from official sonos integration in home assistant:

![image](https://github.com/johanfrick/custom-sonos-card/assets/43878548/e621cfaf-755c-45f1-b178-593492670555)

**Reply by punxaphil** [reply]
I thought the request was about "sub volume and on/off switch,
surrounds volume,
night sound,
speech enhancement" ?

**Reply by bednim** [reply]
my bad, here it is:

![image](https://github.com/johanfrick/custom-sonos-card/assets/43878548/fa448784-fec3-4f57-b7d6-2fbf29129680)

**Reply by punxaphil** [reply]
Alright, I can take a look. It will probably take some time. Not too much spare time for this. Some [<img src="https://github.com/johanfrick/custom-sonos-card/assets/26582898/96fd322f-2fd7-44f1-8fd0-d58cc7db7957" width="100"> ](https://www.buymeacoffee.com/punxaphil) might speed me up though 😉

**Comment by punxaphil** [comment]
Turned out to be quick for me. Fixed in https://github.com/johanfrick/custom-sonos-card/releases/tag/v6.11.0.

Example 1 for TV:
![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/040c7231-715b-4a25-865a-1538114dda18)

Example 2 for group with standard players:
![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/e754b964-4a29-440b-b344-4095a03cefd1)
